api.yaml

```diff
@@ -322,6 +322,76 @@ paths:
               #! end !#
               "bodyPassthrough": true
             }
+
+  /ops/pgsql/v1/schema/version/stepUp:
+    get:
+      summary: "Migrate database schema one version up"
+      description: "Allows to perform schema migrations separately from deployments"
+      responses:
+        200:
+          description: "Resulting Schema Version"
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/SchemaVersion'
+      x-transportd:
+        backend: app
+        enabled:
```

Comment:
question: are there authorization plugins or ASAP like plugins `validateheaders` we should be enabling here or would that best be done in bitbucket?

Reply:
None of the auth logic is present in Github as it is specific to our deployment.